### ISSUE TITLE
aft-invoke-customizations : support nested OU for invocation

### ISSUE THREAD

**Issue by wellsiau-aws** [aws-ia/terraform-aws-control_tower_account_factory#126]
**Describe the outcome you'd like**

`aft-invoke-customizations` to support invocation using nested OU as the target for include / exclude.

**Is your feature request related to a problem you are currently experiencing? If so, please describe.**

I want to invoke customization using `include` with type `ou` as documented in here [1]

```
{
  "include": [
    {
      "type": "ous",
      "target_value": [
        "Sub-Custom"
      ]
    }
  ]
}
```

At the moment, `aft-invoke-customizations` only supports child OU under the root, as shown in here [2]. While the invocation was successful, but the results shown no output target accounts under that OU:

```
{
  "include": [
    {
      "type": "ous",
      "target_value": [
        "Sub-Custom"
      ]
    }
  ],
  "targets": {
    "number_pending_accounts": 0,
    "pending_accounts": []
  }
}
```
 

**Additional context**

[1] https://docs.aws.amazon.com/controltower/latest/userguide/aft-account-customization-options.html#aft-re-invoke-customizations
[2] https://github.com/aws-ia/terraform-aws-control_tower_account_factory/blob/1a13ce0c598452cc0bc75560cec45e3cccd135b9/sources/aft-lambda-layer/aft_common/aft_utils.py#L729

**Comment by aandsco** [comment]
+1

**Comment by balltrev** [comment]
Thanks for bringing this to our attention @aandsco @wellsiau-aws , I've gone ahead and made a backlog to address this feature request!

**Comment by thethomp** [comment]
We just spent some time troubleshooting this and were confused as well, glad we found this issue and know that it is not supported. Would be great to have this fixed!

**Comment by jakebark** [comment]
+1

**Comment by affanc** [comment]
Hi Team, Do we know when this feature request would be resolved?

**Comment by stumins** [comment]
@wellsiau-aws @fjromerom 

Hi all,

We just released [1.6.5](https://github.com/aws-ia/terraform-aws-control_tower_account_factory/releases/tag/1.6.5) which resolves this issue. The `aft-invoke-customizations` step function now supports targeting nested OUs using the format “OU Name (ou-id-1234)”
